server/plugin/webhook.go

```diff
@@ -908,6 +908,19 @@ func (p *Plugin) postPullRequestReviewCommentEvent(event *github.PullRequestRevi
 		return
 	}
 
+	post := &model.Post{
+		UserId:  p.BotUserID,
+		Type:    "custom_git_pr_comment",
+		Message: newReviewMessage,
+	}
+
+	repoName := strings.ToLower(repo.GetFullName())
+	commentID := event.GetComment().GetID()
```

Comment:
Note that these are quick functions that do not do network calls, in case this was moved up here for performance reasons